operationsgateway_api/src/routes/ingest_data.py

```diff
@@ -32,6 +36,19 @@
     summary="Submit a HDF file for ingestion into MongoDB",
     response_description="ID of the record document that has been inserted/updated",
     tags=["Ingestion"],
+    response_model=SubmitHDFResponse,
+    responses={
+        201: {
+            "description": "Created and inserted new record with warning",
```

Comment:
```suggestion
            "model": SubmitHDFResponse,
            "description": "Created and inserted new record with warning",
```

Having looked at https://fastapi.tiangolo.com/advanced/additional-responses/#additional-response-with-model, it seems like you can explicitly identify the model used for the status code, which would let us get the schema for both the 201 and 200 case (in the screenshots it only shows for 200 - I assume it defaults to the lowest code when set by `response_model` on line 39?)